test/e2e_v2.t.sol

```diff
@@ -115,3 +115,54 @@ contract TestEndToEndDeploymentV2 is Test {
         // }
     }
 }
+
+contract TestEndToEndDeploymentV2Fuzzing is Test {
+    DeployFactoryV2 public deployFactoryV2;
+    DeployTokenV2 public deployTokenV2;
+    address owner = address(0xB0b);
+    TokenFactoryV2 public tokenFactory;
+
+    function setUp() public {
+        deployFactoryV2 = new DeployFactoryV2();
+        deployTokenV2 = new DeployTokenV2();
+
+        tokenFactory = new TokenFactoryV2{salt: keccak256("TOKEN_FACTORY_SALT")}(owner);
+    }
+
+    function testFuzz_DeployWithAirdrop(uint8 addressCount) public {
+        // @spec Bound the address count to 0->255 (uint8)
+        vm.assume(addressCount > 0 && addressCount <= 200);
```

Comment:
good instinct, but could you remove this assume statement? we should assert it sends tokens _just_ to the owner, if a 0 array is passed